.ci/install_runtime.sh

```diff
@@ -49,26 +49,36 @@ if [ -e "${NEW_RUNTIME_CONFIG}" ]; then
 	runtime_config_path="${NEW_RUNTIME_CONFIG}"
 fi
 
-if [ "$KATA_HYPERVISOR" = "acrn" ]; then
-	echo "Enable acrn configuration.toml"
-	sudo mv "${PKGDEFAULTSDIR}/configuration-acrn.toml" "${PKGDEFAULTSDIR}/configuration.toml"
-fi
+enable_hypervisor_config(){
+	local path=$1
+	sudo mv "$path" "${PKGDEFAULTSDIR}/configuration.toml"
 
-if [ "$KATA_HYPERVISOR" = "firecracker" ]; then
-	echo "Enable firecracker configuration.toml"
-	sudo mv "${PKGDEFAULTSDIR}/configuration-fc.toml" "${PKGDEFAULTSDIR}/configuration.toml"
-	sudo sed -i '/jailer_path/d' "${PKGDEFAULTSDIR}/configuration.toml"
-fi
+}
+
+case "${KATA_HYPERVISOR}" in
+	"acrn")
+		enable_hypervisor_config "${PKGDEFAULTSDIR}/configuration-acrn.toml"
+		;;
+	"cloud-hypervisor")
+		enable_hypervisor_config "${PKGDEFAULTSDIR}/configuration-clh.toml"
+		;;
+	"firecracker")
+		enable_hypervisor_config "${PKGDEFAULTSDIR}/configuration-fc.toml"
+		sudo sed -i '/jailer_path/d' "${PKGDEFAULTSDIR}/configuration.toml"
```

Comment:
jailer is now [disabled by default](https://github.com/kata-containers/runtime/blob/master/cli/config/configuration-fc.toml.in#L21), so I think we can remove this line, right?